src/main.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react'
```

Comment:
There is one more usage of this in `ErrorBoundary.tsx`. Could you switch this to direct import as well?